tests/php/Extension/FluentExtensionTest.php

```diff
@@ -176,7 +176,7 @@ public function testLocalisedMixSorting()
 
             // Sort by the NonLocalisedSort field first then the LocalisedField second both in ascending order
             // so the result will be opposite if the order of the columns is not maintained
-            $objects=MixedLocalisedSortObject::get()->sort(
+            $objects=MixedLocalisedSortObject::get()->orderBy(
```

Comment:
Fixes errors trying to run sql through `sort()` - a new method `orderBy()` was introduced to handle sorting data via raw SQL